internal/usage/usage.go

```diff
@@ -45,7 +45,7 @@ const (
 	Forever                                  = "Option that acknowledges an alert 'forever'. You can't set both the forever option and the until option in the same command."
 	Status                                   = "State of the alert. Valid values include TRACKING, OPEN, CLOSED, and CANCELLED."
 	Until                                    = "ISO 8601-formatted time until which the alert is acknowledged. This command returns this value if a MongoDB user previously acknowledged the alert. After this date, the alert becomes unacknowledged."
-	ConnectionStringType                     = "Type of connection string. If you specify 'private', this option retrieves the connection string for the network peering endpoint. This value defaults to 'standard'."
+	ConnectionStringType                     = "Type of connection string. If you specify 'private', this option retrieves the connection string for the network peering endpoint. If you specify 'privateEndpoint', this option retrieves the shard optimized connection strings for the private endpoints. This value defaults to 'standard'."
```

Comment:
While we're here, it looks like the "This value defaults to "standard"" gets added automatically to the docs files, so it displays twice. Can we remove it here? If it will hold you up with reviews I can open a ticket to do another time, not urgent and unrelated to your change

Reply:
I will update it, thanks!

Reply:
updated